rdg_datasets/empty_graph/storage_format_version_6/part_vers00000000000000000001_rdg_node00000

```diff
@@ -0,0 +1 @@
+{"kg.v1.edge_entity_type_id_array":"edge_entity_type_id_array-fppqocT7GcMP","kg.v1.edge_entity_type_id_dictionary":[[0,[]]],"kg.v1.edge_entity_type_id_name":[],"kg.v1.edge_property":[],"kg.v1.node_entity_type_id_array":"node_entity_type_id_array-kAHpu9DnoVDI","kg.v1.node_entity_type_id_dictionary":[[0,[]]],"kg.v1.node_entity_type_id_name":[],"kg.v1.node_property":[],"kg.v1.part_property_files":[["mirror_nodes_0","mirror_nodes_0-SLdj9k98LDA2"],["master_nodes_0","master_nodes_0-23jWTL2DMULR"],["host_to_owned_global_node_ids","host_to_owned_global_node_ids-hHqPT6iy7kjx"],["host_to_owned_global_edge_ids","host_to_owned_global_edge_ids-Q4m0LaiObIVM"],["local_to_user_id","local_to_user_id-EOOFpx08vFat"],["local_to_global_id","local_to_global_id-xT6MW7VcRDzN"]],"kg.v1.part_property_meta":{"cartesian_grid":[0,0],"is_incoming_edge_cut":false,"is_outgoing_edge_cut":false,"magic":1266713220,"max_global_node_id":18446744073709551615,"num_edges":0,"num_global_edges":0,"num_global_nodes":0,"num_nodes":0,"num_owned":0,"policy_id":0,"transposed":false},"kg.v1.partition_topology_metadata":{"kg.v1.partition_topology_metadata_entries":[{"edge_condensed_type_id_map_present":false,"edge_condensed_type_id_map_size":0,"edge_index_to_property_index_map_present":false,"edge_sort_state":"kAny","node_condensed_type_id_map_present":false,"node_condensed_type_id_map_size":0,"node_index_to_property_index_map_present":false,"node_sort_state":"kAny","num_edges":0,"num_nodes":0,"path":"topology-dyqVp74lR46R","topology_state":"kCSR","transpose_state":"kNo"}],"kg.v1.partition_topology_metadata_entries_size":1},"kg.v1.storage_format_version":5,"kg.v1.unstable_storage_format":false}
```

Comment:
this rdg is `kg.v1.storage_format_version":5`, please put it in a `storage_format_version_5` directory

apologies, v6 is just making its way into the codebase as we speak